File: 75-sort-colors/sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int count0=0,count1=0,count2=0;
        for(int i:nums)
        {
            if(i==0)
            {
                count0++;
            }
            else if(i==1)
            {
                count1++;
            }
            else 
            {
                count2++;
            }
        }
        int i=0;
        while(i<nums.size())
        {
            while(count0>0)
                {
                    nums[i++]=0;
                    count0--;
                }
            while(count1>0)
            {
                nums[i++]=1;
                count1--;
            }
            while(count2>0)
            {
                nums[i++]=2;
                count2--;
            }
        }
    }
};

File: 75-sort-colors/sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        c = dict()
        for i in nums:
            if i not in c:
                c[i] =1
            else:
                c[i] = c[i] +1
        k=0
        for i in sorted(c.keys()):
            while c[i]>0:
                nums[k] = i
                k+=1
                c[i]-=1

        